### PR TITLE
#1477 - Document creation from external URL

### PR DESCRIPTION
**Pull request by dsuren1**
### Description
This PR adds new feature to upload document via external URL

### Issue
- #1477 
image

### CODE REVIEW

**Review by allyoucanmap — CHANGES_REQUESTED** [review]
- [ ] clicking multiple time on the add url generates this error in console
image
- [ ] added a unsupported format such as .svg. The ext is disabled but the user does not get warning and the upload is disabled
image

Note if we try to upload via file we get the warning, the behaviour should be the same
image

for dataset if the upload is incomplete we add an exclamation mark and a small description of what missing

image

so if possible we should have all warning set and visible, in addition if not too complex I would also align the size of the extension select input with the other

image

**Comment by dsuren1** [comment]
**Updated**
Screenshot 2023-07-06 at 6 51 38 PM

File: geonode_mapstore_client/static/mapstore/gn-translations/data.en-US.json [hunk 1]
```diff
@@ -270,6 +270,9 @@
             "invalidUploadMessageErrorTooltip": "The upload process failed. Please check the validity of the file and try to upload it again",
             "completeUpload": "Complete the upload",
             "selectFiles": "Select files...",
+            "addUrl": "Add URL...",
+            "duplicateUrl": "URL is already present in the list of uploads",
```
Comment (allyoucanmap) [inline geonode_mapstore_client/static/mapstore/gn-translations/data.en-US.json:274]: Please update all translations with:
`This URL is already present in the list of uploads`

**Review by allyoucanmap — CHANGES_REQUESTED** [review]
@dsuren1 Sent a commit with some style updated so please pull the latest changes

image

image

I found this problem

- [ ] issue with not supported extensions
  - add a new document url
  - add an svg eg https://development.demo.geonode.org/static/mapstore/img/geonode-logo.svg
  - upload is blocked because the extension is not supported but the extension selection is still available even if it does not have effect
https://github.com/GeoNode/geonode-mapstore-client/assets/19175505/98fb56ae-fec5-453b-91f3-b0e43eca6a2f

**Comment by dsuren1** [comment]
> I found this problem
> 
> * [ ]  issue with not supported extensions
>   * add an svg eg https://development.demo.geonode.org/static/mapstore/img/geonode-logo.svg
>   * upload is blocked because the extension is not supported but the extension selection is still available even if it does not have effect

This issue is happening after the style change commit, looks like disabled class was removed. I will add it back

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**